src/components/testCaseLanding/qiCore/TestCaseList.tsx

```diff
@@ -227,6 +229,37 @@ const TestCaseList = (props: TestCaseListProps) => {
       });
   };
 
+  const exportTestCase = async (selectedTestCase: TestCase) => {
+    try {
+      abortController.current = new AbortController();
+      const { ecqmTitle, model, version } = measure ?? {};
+      const exportData = await testCaseService?.current.exportTestCase(
+        measure?.id,
+        selectedTestCase.id,
+        abortController.current.signal
+      );
+      const url = window.URL.createObjectURL(exportData);
+      const link = document.createElement("a");
+      link.href = url;
+      link.setAttribute(
+        "download",
+        `${ecqmTitle}-v${version}-${getModelFamily(model)}-TestCases.zip`
+      );
+      document.body.appendChild(link);
+      link.click();
```

Comment:
might be able to replace 241-249 with the `file-saver` utility that offers a `saveAs` function for file downloads